docs/project/faq.md

```diff
@@ -139,8 +139,8 @@ one.
 
 ## Why build Carbon?
 
-See the [project README](#why-build-carbon) for an overview of the motivation
-for Carbon. This section dives into specific questions in that space.
+See the [project README](#why-build-carbon) for an overview of the
+motivation for Carbon. This section dives into specific questions in that space.
```

Comment:
When I format this paragraph as a full line:

```
See the [project README](#why-build-carbon) for an overview of the motivation for Carbon. This section dives into specific questions in that space.
```

And then execute `pre-commit run` it formats it with "motivation" in the first line.

Figuring out why there is a difference here could help a lot.

Reply:
I assume this was written after seeing the issue in https://github.com/carbon-language/carbon-lang/pull/1825. The formatting changes are as a consequence of the other edits in that PR. This PR drops the link fix that is at the heart of the other PR, and so there is no need to reformat. We use pre-commit specifically so that we can get formatting issues addressed prior to merge, without separate format PRs such as this one.

i.e., I think this change is the result of a misunderstanding of the formatting issue on #1825.